js/interface.js

```diff
@@ -44,13 +44,14 @@ function oembed(options) {
       var notSupported = ['video', 'link'].indexOf(response.type) === -1;
 
       if (response.thumbnail_url) {
-        if (!response.width) {
+        if (!response.width || response.thumbnail_width) {
           response.width = response.thumbnail_width;
         }
 
-        if (!response.height) {
+        if (!response.height || response.thumbnail_height) {
           response.height = response.thumbnail_height;
         }
+
```

Comment:
This empty line shouldn't be here. Is this empty line added manually or added by the formatter/linter?

Reply:
it was added due to formatter and removed this empty line.

Reply:
actually, while fliplet-widget-video-online repo bymistakly added another extension. So that empty line was added. I uninstall it but forgot to remove these empty lines while raising the PR. I will take care of these things next time while raising any PR.